docs/bioVince.md

```diff
@@ -0,0 +1,3 @@
+<h1>Vince</h1>
+* Bio
+  -Line 2
```

Comment:
Maybe in another pull request, we can edit this file and make this rendered into a second bulleted list item! Hold on for more details on this when we work from the command line!